__tests__/pet.test.js

```diff
@@ -0,0 +1,292 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  let pet;
```

Comment:
You can defined pet, and the beforeEach() outside this describe() block, moving it to the global scope - which means you don't need to keep repeating a beforeEach statement per describe().